### PR TITLE
Add ETH Belgrade 2024 to community events

### PR DESCRIPTION
**Pull request by develo-pera**
Add ETH Belgrade 2024 to the list

## Description

Added ETH Belgrade 2024 website, description and dates

### CODE REVIEW

**Comment by develo-pera** [comment]
@wackerow who can I ask to check this PR? Thank you

File: src/data/community-events.json
Comment (wackerow) [inline src/data/community-events.json:23]: @minimalsm @corwintines We haven't been enforcing this, so not going to block this PR, but separately we should decide if we want to keep the "sponsor" field going forward or to just remove, since it clearly hasn't been enforced in the current list of events.

File: src/data/community-events.json
Comment (wackerow) [inline src/data/community-events.json:20]:
```suggestion
  {
```

File: src/data/community-events.json
Comment (corwintines) [reply src/data/community-events.json:23]: I created a PR here removing it and typing this data: https://github.com/ethereum/ethereum-org-website/pull/12000

Going to bring this in here, and will fix anything that needs to be done in the above PR.

**Review by corwintines — APPROVED** [review]

**Pull request merged**
